main.nf

```diff
@@ -929,6 +901,7 @@ process fetch_MTseqs {
 
     output:
         set group, id, file ("${id}_mito.bam"), file("${id}_mito.bam.bai") into mutserve_bam, eklipse_bam
+        set group, id, file ("${id}_mito.bam"), file("${id}_mito.bam.bai") into qc_mito_bam
```

Comment:
since this output channel is identical to the above, you can just add it to the above output.

into mutserve_bam, eklipse_bam, qc_mito_bam

Reply:
Fixed!